manifest/net.kazu0617/SessionTweaks/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "SessionTweaks",
+	"id": "net.kazu0617.sessiontweaks",
+	"description": "Add these 3 buttons, 'Spawn' as spawn session orb, 'Open' as join session without focusing' and 'Copy' as copies session uri in Contacts tab.",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/kazu0617/SessionTweaks",
```

Comment:
I'd suggest adding the tag "world" as a something this would be findable by.
`"tags": ["world"],` since it doesn't appear in the name or description.

Reply:
Thanks for the suggestion. updated it.